hibernate-core/src/main/java/org/hibernate/boot/model/naming/ObjectNameNormalizer.java

```diff
@@ -38,8 +38,8 @@ public Identifier normalizeIdentifierQuoting(String identifierText) {
 		return database().toIdentifier( identifierText );
 	}
 
-	protected Database database() {
-		return getBuildingContext().getMetadataCollector().getDatabase();
+	public Database database() {
+			return getBuildingContext().getMetadataCollector().getDatabase();
```

Comment:
Did you mean to increase the indent here?

Reply:
Nope, this method actually shouldn't even change in this version - thanks for catching it!